core/src/stages/compute_ik.cpp

```diff
@@ -314,6 +314,11 @@ void ComputeIK::compute() {
 
 		link = scene->getCurrentState().getRigidlyConnectedParentLinkModel(ik_pose_msg.header.frame_id);
 
+		if (!link) {
+			RCLCPP_WARN_STREAM(
+			    LOGGER, fmt::format("ik frame '{}' is not rigidly connected to any link", ik_pose_msg.header.frame_id));
```

Comment:
Would this be more precise?
```suggestion
			    LOGGER, fmt::format("ik frame '{}' is not connected to any parent link", ik_pose_msg.header.frame_id));
```

Reply:
As I said, I expect this to return NULL if and only if frame doesn't exist. Do you have other evidence?

Reply:
@sjahr: I'm still curious to learn about your concrete issue. 
In which case does `getRigidlyConnectedParentLinkModel` returns NULL, which is not handled before by `knowsFrameTransform`?

Reply:
Sorry, I cannot share more info on this at the moment :disappointed: I'll try debugging it further and hopefully figure out what the real source behind this is.

Reply:
I'm not expecting a detailed error analysis, but ideally a code snippet reproducing your segfault - I assume you observed a segfault in some code.

Reply:
Yes, unfortunately I don't have access to that code. If you'd prefer that I can close this PR for now and come back later with more solid changes?